packages/halo-theme/src/fonts.less

```diff
@@ -10,7 +10,7 @@
 @font-face {
   font-family: 'Proxima Nova Fin';
   font-style: normal;
-  font-weight: 500;
+  font-weight: 500 600;
```

Comment:
I think it's a bit odd with this line.

Reply:
I see that it supports multiple value. https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-weight

Reply:
Too weird, I didn't see this syntax before, and when I try with my browser, e.g. Edge Chrome and Safari. It shows the same warning like 'invalid property value'.

Reply:
as we discussed, it's `font-weight` in @font-face. It's not font-weight that you can set to element.